libsql_test.go

```diff
@@ -108,8 +109,8 @@ func (db Database) sync() {
 }
 
 type Table struct {
-	name string
 	db   Database
+	name string
```

Comment:
I don't think memory usage of tests justify less readable code.

Reply:
fair enough. reverting this.